php-8.2.yaml

```
@@ -1,11 +1,13 @@
 package:
-  name: php
+  name: php-8.2
   version: 8.2.10
-  epoch: 0
+  epoch: 1
   description: "the PHP programming language"
   copyright:
     - license: PHP-3.01
   dependencies:
+    provides:
+      - php=${{package.version}}-r${{package.epoch}}
```

```suggestion
      - php=${{package.version}}
```

I don't think this is needed? but I'm also php illiterate so feel free to ignore